src/asset/asset.h

```diff
@@ -9,12 +9,13 @@
 
 struct AssetID {
     AssetID(const char *);
-    AssetID(u64 id): id(id) {}
-    AssetID(): id(NONE()) {}
+    AssetID(u64 id): id(id), name("NO_NAME") {}
+    AssetID(): id(NONE()), name("NO_NAME") {}
 
     static AssetID NONE() { return 0xFFFFFFFF; }
 
     u64 id;
+    const char *name;
```

Comment:
I'm a bit weary on adding members to the ID, I would think of moving this to the asset. Because we now need to store the actual strings, before we could get away with storing only the hashed version.

Reply:
Yes, I think we'll skip this entirely for now and instead just keep names on the entities since it basically sidesteps the hashes. It's still needed in some way somewhere down the line since we need to be able to recognize one sound from another if we implement a cool mixer or w/e. (Maybe named channels?)